tests/trainer/test_trainer.py

```diff
@@ -403,6 +403,7 @@ def test_model_freeze_unfreeze():
         assert param.requires_grad
 
 
+@pytest.mark.xfail(reason="FIXME(@carmocca): this test wasn't running and is now broken")
```

Comment:
Since the monitor does not exist and it did not fail, no checkpoints were written so the assertions did not run

Meaning, the test is broken in master.

Will work on fixing it in a separate PR